app/src/main/java/com/example/android/cats/ui/detail/DetailViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.android.cats.ui.detail
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.android.cats.repositories.CatsRepository
+import com.example.android.cats.repositories.network.pojo.CatItem
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.launch
+
+class DetailViewModel : ViewModel() {
+
+    private val repository: CatsRepository by lazy { CatsRepository() }
```

Comment:
Why do you need two versions of repository? I hope it was done just to simplify your simple code base.